firmware/application/ui/ui_spectrum.cpp

```diff
@@ -390,19 +390,23 @@ uint32_t filter_bandwidth_for_sampling_rate(int32_t sampling_rate) {
                                        // BW 500k...750kHz, ex. 500kHz (fs = 8 x BW = 4Mhz), BW 600kHz (fs = 4,8Mhz), BW 750 kHz (fs = 6Mhz).
             return 2'500'000;          // In some IC, MAX2837 appears as 2250000, but both work similarly.
 
-        case 8'800'000:  // BW capture 1,1Mhz fs = 8 x 1,1Mhz = 8,8Mhz. (1Mhz showed slightly higher noise background).
+        case 8'000'000:  // BW capture 1Mhz fs = 8 x 11Mhz = 8Mhz. (1Mhz showed slightly higher noise background).
```

Comment:
Please correct comment.  (It should say 1MHz, not 11MHz.  Also the sentence afterwards implies that 1MHz isn't good?)

Reply:
Thanks for checking , 
Yes you are right , it should say 1Mhz. It seems I deleted the coma , but not the additional 1 digit . 
Let me ammend it , and I will inform you .

Reply:
Solved, thanks @NotherNgineer